web/src/components/filings-calendar/FilingsCalendarSingleGrid.tsx

```diff
@@ -41,10 +43,51 @@ export const FilingsCalendarSingleGrid = (props: Props): ReactElement => {
     );
   });
 
-  const firstTwoFilings = thisMonthFilings.slice(0, NUM_OF_FILINGS_ALWAYS_VIEWABLE);
+  const visibleFilings = thisMonthFilings.slice(0, NUM_OF_FILINGS_ALWAYS_VIEWABLE);
   const remainingFilings = thisMonthFilings.slice(NUM_OF_FILINGS_ALWAYS_VIEWABLE);
   const isOnCurrentYear = getCurrentDate().year().toString() === props.activeYear;
 
+  const licenseEvent = getLicenseCalendarEvent(
+    parseDate(props.userData?.licenseData?.expirationISO).add(4, "year").toISOString(),
```

Comment:
Why are we adding 4 years? And why adding it here, in a component, and not in the function? That absolutely seems like domain logic, not render logic.

If the expiration date we're getting from the backend is off by 4 years, shouldn't we even do this on the backend before saving it to begin with?